### PR TITLE
RavenDB-16539 - Create new license attribute: readOnlyCertificates

### PR DESCRIPTION
**Pull request by grisha-kotler**

### CODE REVIEW

File: src/Raven.Client/Exceptions/Commercial/LimitType.cs [hunk 1]
```diff
@@ -65,6 +65,9 @@ public enum LimitType
         AdditionalAssembliesFromNuGet,
         
         [Description("Monitoring Endpoints")]
-        MonitoringEndpoints
+        MonitoringEndpoints,
+
+        [Description("Read Only Certificates")]
```
Comment (ppekrol) [inline src/Raven.Client/Exceptions/Commercial/LimitType.cs]: Read-only

File: src/Raven.Server/Commercial/LicenseManager.cs [hunk 2]
```diff
@@ -1324,6 +1325,21 @@ public void AssertCanUseMonitoringEndpoints()
...
+            if (certificate.Permissions.All(x => x.Value == DatabaseAccess.Read) == false)
+                return;
+
+            if (IsValid(out var licenseLimit) == false)
+                throw licenseLimit;
+
+            if (LicenseStatus.HasReadOnlyCertificates)
+                return;
+
+            const string details = "Your current license doesn't include the read only certificates feature";
```
Comment (ppekrol) [inline src/Raven.Server/Commercial/LicenseManager.cs]: read-only

File: src/Raven.Studio/wwwroot/App/views/shell/about.html [hunk 3]
```diff
@@ -155,6 +158,7 @@ <h4>Security</h4>
                     <div class="name">Certificates</div><div class="value"><i class="icon-checkmark"></i></div>
                     <div class="name">Encryption in transit</div><div class="value"><strong>TLS 1.2 & X.509</strong></div>
                     <div class="name">Storage encryption</div><div class="value"><i data-bind="attr: { class: licenseAttribute('HasEncryption') }"></i></div>
+                    <div class="name">Read Only Certificates</div><div class="value"><i data-bind="attr: { class: licenseAttribute('HasReadOnlyCertificates') }"></i></div>
```
Comment (ppekrol) [inline src/Raven.Studio/wwwroot/App/views/shell/about.html]: Read-only

Comment (ppekrol) [reply src/Raven.Studio/wwwroot/App/views/shell/about.html]: Move it after the 'Certificates' maybe?

File: src/Raven.Client/Exceptions/Commercial/LimitType.cs (hunk 1 above)
Comment (grisha-kotler) [reply src/Raven.Client/Exceptions/Commercial/LimitType.cs]: done

File: src/Raven.Server/Commercial/LicenseManager.cs (hunk 2 above)
Comment (grisha-kotler) [reply src/Raven.Server/Commercial/LicenseManager.cs]: done

File: src/Raven.Studio/wwwroot/App/views/shell/about.html (hunk 3 above)
Comment (grisha-kotler) [reply src/Raven.Studio/wwwroot/App/views/shell/about.html]: done

**Comment by ppekrol** [comment]
@grisha-kotler please check failing tests

**Review by ppekrol — APPROVED** [review]

**Pull request merged**